Taskfile.yml

```diff
@@ -6,8 +6,8 @@ env:
   GOCOVERDIR: tmp/cover
 
 vars:
-  SHARD_INDEX: 1
-  SHARD_TOTAL: 1
+  SHARD_INDEX: 0
```

Comment:
Why did we move it back?

Reply:
What do you mean by moving it back?

Reply:
From 1 to 0.

Reply:
https://github.com/FerretDB/FerretDB/pull/4480/files#diff-a7c8b37b0eee875782802ed67486da60bee4d47b8800407666e9c365a6715808R77

Reply:
Not allowing `--run` or `-skip` to be used at the same time with `--shard`. This simplifies the code.

Reply:
I think the real answer is that this PR also ports things from v2